src/parser/core/changelog.tsx

```diff
@@ -23,6 +23,13 @@ export const changelog: ChangelogEntry[] = [
 		</>,
 		contributors: [CONTRIBUTORS.HINT, CONTRIBUTORS.AYA],
 	},
+	{
+		date: new Date('2023-07-21'),
+		Changes: () => <>
+			Fixed a bug that was breaking raid buff window detection for players in a party with a Summoner that did not use <DataLink action="RADIANT_AEGIS" showIcon={false} />.
```

Comment:
I know we're not wildly consistent on this but I think all the other Core changelogs are written in present tense (with one exception directly below this particular change).

Reply:
We're kind of all over the place in Core too, some of the older ones have past tense as well. We'll leave it for now.